### DISCUSSION TITLE
devtools::install() not working

### DISCUSSION THREAD

**Discussion by lucyparker25** [IBAHCM/RPiR#299] [Archive] [answered]
Hi, I keep running into this error when I try to run devtools::install():
Error: HTTP error 403.
  API rate limit exceeded for 86.130.43.8. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

  Rate limit remaining: 0/60
  Rate limit reset at: 2021-02-15 12:42:32 UTC

  To increase your GitHub API rate limit
  - Use `usethis::browse_github_pat()` to create a Personal Access Token.
  - Use `usethis::edit_r_environ()` and add the token as `GITHUB_PAT`.
In addition: There were 27 warnings (use warnings() to see them)

I have tried restarting R but it doesn't make a difference! Is this something you have seen before? I know there are suggestions in the error message but I don't want to strat clicking stuff when I don't know what it means. Thanks.

**Comment by soniamitchell** [comment, marked as answer]
Hi @lucyparker25, could you try this? https://ibahcm.github.io/RPiR/articles/pages/install_github.html#add-a-pat

**Reply by lucyparker25** [reply]
Thank you, that fixed the issue!

**Comment by mikatakahashi** [comment]
Hi @soniamitchell 
I also get long error message when I try devtool::install(). Do you know what is happening? I restarted R but it did not help.
<img width="1435" alt="Screen Shot 2021-02-16 at 12 06 23" src="https://user-images.githubusercontent.com/77616349/108060866-65f1ba00-704f-11eb-9225-b3772b88e3a4.png">

**Reply by soniamitchell** [reply]
Hey @mikatakahashi, you have a different issue, so I think we should start a new thread (maybe @richardreeve could move this because I don't know how).

Your issue is the the installation of MikaTakahashiBCI is being skipped. Could you try changing your `Remotes:` field in your description file? Replace line 21 (`IBAHCM/BiodivMika,`) with `IBAHCM/MikaTakahashiBCI,`, and try again? 👍
